### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by jaredmcphillen**
Created a dockerfile and documentation to use. Ease install, and contain all changes within a separate environment. e.g. python 3.8, pip packages, etc.

### CODE REVIEW

**Comment by palemieux** [comment]
@jaredmcphillen I suggest moving the Dockerfile to a dedicated `container/centos` directory since it is super specific to that OS. Ok?

**Comment by jaredmcphillen** [comment]
Yes.

**Comment by palemieux** [comment]
@jaredmcphillen Great. I will merge the PR once you update it.

**Comment by jaredmcphillen** [comment]
@palemieux made the requested changes.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,29 @@ Example (creates a supplementatl delivery where the assets are symlinked to the
 
 `python src/main/python/repkl/cli.py --action symlink --ov delivery/CPL_bb2ce11c-1bb6-4781-8e69-967183d02b9b delivery/CPL_0b976350-bea1-4e62-ba07-f32b28aaaf30.xml new_delivery/`
 
+## Docker
```
Comment (palemieux) [inline README.md]:
```suggestion
## CentOS Docker Container 
```

**Review by palemieux — CHANGES_REQUESTED** [review]

**Review by palemieux — APPROVED** [review]

**Pull request merged**